ai-decision-services/ai-ad-dpp-engine/prepare-config-data-and-app/prepare-config-data-and-app.md

```diff
@@ -0,0 +1,217 @@
+
+Lab 3: Preparation code, config, library and datasets
+===
+
+Fundamentally, you will need the following things ready for setting up the whole infrastructure:
+
+*   Orchestration configuration
+*   Driver code (Orchestrator) - from our repo
+*   Training/Inferencing datasets
+*   Library (Archive.zip)
+
+## Orchestration Configuration
+
+This is a JSON file that defines the DPP workflow. The configuration is composed of the following 5 parts:
```

Comment:
When referencing an abbreviation for the first time in a given lab, include it within brackets like so -
'This is a JSON file that defines the Data Pre-Processing (DPP) workflow......'